{"uid":"738938a7262cdc1d","name":"APP-2763: Ошибка при создании борды с длинным заголовком custom field: Select","fullName":"tests.test_backend.board.board_validation_test#test_create_board_with_long_custom_field_title","historyId":"faa5dde8e7f920bda1d8a3851ce59ecc","time":{"start":1749744412451,"stop":1749744412705,"duration":254},"status":"passed","statusMessage":"XPASS Известный баг: длинный заголовок custom field без пробелов не влезает в тултип (APP-2763)","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"delete_output_dir","time":{"start":1749744399429,"stop":1749744399429,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"_verify_url","time":{"start":1749744399429,"stop":1749744399429,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"pytestconfig","time":{"start":1749744399429,"stop":1749744399429,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"base_url","time":{"start":1749744399429,"stop":1749744399429,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"owner_client","time":{"start":1749744399429,"stop":1749744400163,"duration":734},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"temp_space","time":{"start":1749744400163,"stop":1749744401184,"duration":1021},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"temp_project","time":{"start":1749744401184,"stop":1749744402659,"duration":1475},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"temp_board","time":{"start":1749744405216,"stop":1749744405548,"duration":332},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"status":"passed","statusMessage":"XPASS Известный баг: длинный заголовок custom field без пробелов не влезает в тултип (APP-2763)","steps":[{"name":"Отправка запроса с custom field title длиннее 50 символов","time":{"start":1749744412451,"stop":1749744412705,"duration":254},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"Проверка, что API вернул 400 – ошибка валидации длины заголовка","time":{"start":1749744412705,"stop":1749744412705,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":2,"attachmentsCount":0,"hasContent":true},"afterStages":[{"name":"temp_space::0","time":{"start":1749744437355,"stop":1749744438003,"duration":648},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"bug","value":"APP-2763"},{"name":"tag","value":"@pytest.mark.xfail(reason='Известный баг: длинный заголовок custom field без пробелов не влезает в тултип (APP-2763)')"},{"name":"tag","value":"backend"},{"name":"parentSuite","value":"tests.test_backend.board"},{"name":"suite","value":"board_validation_test"},{"name":"host","value":"pkrvmxyh4eaekms"},{"name":"thread","value":"2515-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_backend.board.board_validation_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"field_type","value":"'Select'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":7,"passed":3,"unknown":0,"total":10},"items":[{"uid":"c65d7af8a0db2c3e","reportUrl":"https://vaizcom.github.io/autotests/124//#testresult/c65d7af8a0db2c3e","status":"passed","statusDetails":"XPASS Известный баг: длинный заголовок custom field без пробелов не влезает в тултип (APP-2763)","time":{"start":1749744325113,"stop":1749744325316,"duration":203}},{"uid":"8bccb6a6015c2bf","reportUrl":"https://vaizcom.github.io/autotests/123//#testresult/8bccb6a6015c2bf","status":"skipped","statusDetails":"XFAIL Известный баг: длинный заголовок custom field без пробелов не влезает в тултип (APP-2763)\n\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f6ed7455e50>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749744004389,"stop":1749744004389,"duration":0}},{"uid":"a42102d1a10d218a","reportUrl":"https://vaizcom.github.io/autotests/122//#testresult/a42102d1a10d218a","status":"passed","statusDetails":"XPASS Известный баг: длинный заголовок custom field без пробелов не влезает в тултип (APP-2763)","time":{"start":1749743628874,"stop":1749743629092,"duration":218}},{"uid":"25e9e4498ada8c08","reportUrl":"https://vaizcom.github.io/autotests/119//#testresult/25e9e4498ada8c08","status":"skipped","statusDetails":"XFAIL Известный баг: длинный заголовок custom field без пробелов не влезает в тултип (APP-2763)\n\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fc7b5913490>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749742821499,"stop":1749742821499,"duration":0}},{"uid":"504859d9f9666db5","reportUrl":"https://vaizcom.github.io/autotests/118//#testresult/504859d9f9666db5","status":"skipped","statusDetails":"XFAIL Известный баг: длинный заголовок custom field без пробелов не влезает в тултип (APP-2763)\n\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f9cb88f1f10>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749742668930,"stop":1749742668930,"duration":0}},{"uid":"34be8b5b635d55c0","reportUrl":"https://vaizcom.github.io/autotests/117//#testresult/34be8b5b635d55c0","status":"skipped","statusDetails":"XFAIL Известный баг: длинный заголовок custom field без пробелов не влезает в тултип (APP-2763)\n\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f2b9f21f250>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749742498745,"stop":1749742498745,"duration":0}},{"uid":"4602119c8863811d","reportUrl":"https://vaizcom.github.io/autotests/116//#testresult/4602119c8863811d","status":"skipped","statusDetails":"XFAIL Известный баг: длинный заголовок custom field без пробелов не влезает в тултип (APP-2763)\n\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7efcc7f78ad0>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749741235109,"stop":1749741235109,"duration":0}},{"uid":"c987c22146dacb1b","reportUrl":"https://vaizcom.github.io/autotests/115//#testresult/c987c22146dacb1b","status":"skipped","statusDetails":"XFAIL Известный баг: длинный заголовок custom field без пробелов не влезает в тултип (APP-2763)\n\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f28d1ddcd10>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749740827681,"stop":1749740827681,"duration":0}},{"uid":"823e114efaf2ad8","reportUrl":"https://vaizcom.github.io/autotests/114//#testresult/823e114efaf2ad8","status":"skipped","statusDetails":"XFAIL Известный баг: длинный заголовок custom field без пробелов не влезает в тултип (APP-2763)\n\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f271b110ad0>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749740654369,"stop":1749740654369,"duration":0}}]},"tags":["backend","@pytest.mark.xfail(reason='Известный баг: длинный заголовок custom field без пробелов не влезает в тултип (APP-2763)')"]},"source":"738938a7262cdc1d.json","parameterValues":["'Select'"]}